docs/guides/admin/docs/index.md

```diff
@@ -3,37 +3,27 @@ Opencast {{ opencast_major_version() }} Administration Guide
 
 Welcome to the Opencast Universe! Opencast is an open-source enterprise level lecture recording system. The core of the
 system delivers functionality for scheduling, media encoding, editing and content delivery. For lecture capture,
-Opencast provides capture agent software and third party appliances are available. An awesome community provides new
-features and support.
+Opencast third party appliances are available. An awesome community provides new features and support.
```

Comment:
pyCA is third party?

Reply:
I figured this referenced the internal Capture Agent that Opencast used to have, but yeah, good point. No idea if this meant PyCA.